analysis/interactive.Rmd

```diff
@@ -4,21 +4,265 @@ output:
+Then, users may select the first row in the “Row statistics table 1” and watch the second and third panel automatically regenerate to display the most significant marker gene on the y-axis (“Feature assay plot 1” panel), and as a color scale overlaid on the data points (“Reduced dimension plot 1” panel).
+
+```{r, eval=FALSE}
+# Configure a "Row statistics table" panel (NOTE: we leave all parameters to default)
+rowStatArgs <- rowStatTableDefaults(sce.pbmc, 1)
+
+# Configure a "Feature assay plot" panel
+featAssayArgs <- featAssayPlotDefaults(sce.pbmc, 1)
+featAssayArgs$YAxisRowTable <- "Row statistics table 1"
+featAssayArgs$XAxis <- "Column data"
+featAssayArgs$XAxisColData <- "cluster"
+featAssayArgs$DataBoxOpen <- TRUE
+
+# Configure a "Reduced dimension plot" panel
+redDimArgs <- redDimPlotDefaults(sce.pbmc, 1)
+redDimArgs$Type <- "UMAP"
+redDimArgs$ColorBy <- "Feature name"
+redDimArgs$ColorByRowTable <- "Row statistics table 1"
+redDimArgs$VisualBoxOpen <- TRUE
+initialPanels <- DataFrame(
+    Name = c("Row statistics table 1", "Feature assay plot 1", "Reduced dimension plot 1"),
+    Width = c(4L, 4L, 4L)
+)
+
+# Prepare the app
+app <- iSEE(sce.pbmc, rowStatArgs = rowStatArgs, featAssayArgs = featAssayArgs,
+  redDimArgs = redDimArgs, initialPanels = initialPanels)
+
+# Launch the app
+if (interactive()){
+  runApp(app)
+}
+```
+
+In addition to selecting the top genes by significance, it is also possible to search the table for arbitrary gene names and select any known marker gene.
+Practically, investigating the expression levels of known marker genes is a common approach to assigning cell type labels to inferred clusters.
+
+### Collaborative analysis
+
+Analysis and interpretation of complex data sets typically involves multiple analysts.
+Rather than sharing analysis results via multiple versions of static figures and reports, it is often helpful to share an interactive version of a plot, allowing the other involved parties to independently explore the data in further depth.
+
+With `iSEE`, this can be achieved at any time by clicking on the "Display panel settings" button under the "Diagnostics" tab in the top right corner, copying the displayed code into an R script, and sharing this chunk of code with collaborators.
+However, note that they will also need to have access to a copy of the original `sce` object used for the analysis.
+In that setup, executing the code in the script and launching `iSEE` with the following command will open an instance mimicking what was shown on the screen when the "Display panel settings" button was clicked.
+It is important to note that all `iSEE` applications remain interactive irrespective of their initial configuration; only the initial layout and state of the application is modified from the default setup. 
+
+```{r, eval=FALSE}
+iSEE(sce, redDimArgs = redDimPlotArgs, colDataArgs = colDataPlotArgs,
```

Comment:
This is pretty confusing if `redDimPlotArgs` and friends are never actually created.

Reply:
I've removed the code chunk. Resolving this.

Reply:
Code chunk removed in 147c21c